.gitignore

```diff
@@ -21,7 +21,6 @@ GeneratedKey.snk
 [Rr]eleases/
 x64/
 x86/
-build/
```

Comment:
Because we use `bld`, not `build` for our output, and I like to put build-related scripts and files in the `src\build` directory (see below).